src/app_setup.jl

```diff
@@ -1,14 +1,14 @@
 const appname_module_dict = Dict(
-  :common  => :Common,
-  :fan  => :Fan,
-  :fulton  => :Fulton,
-  :graph  => :Graph,
-  :group  => :Group,
-  :ideal  => :Ideal,
-  :matroid  => :Matroid,
-  :polytope  => :Polytope,
-  :topaz  => :Topaz,
-  :tropical  => :Tropical
+  :common     => :Common,
+  :fan        => :Fans,
+  :fulton     => :Fulton,
+  :graph      => :Graphs,
+  :group      => :Groups,
+  :ideal      => :Ideals,
+  :matroid    => :Matroids,
+  :polytope   => :Polytopes,
+  :topaz      => :Topaz,
+  :tropical   => :Tropical
 )
```

Comment:
Why do we need to rename the modules?

Reply:
```julia
julia> module Polytope
       struct Polytope end
       end
ERROR: invalid redefinition of constant Polytope
Stacktrace:
 [1] top-level scope at REPL[1]:2

```

Reply:
ah yes. We already had this problem discussion ....

btw this would work 
```
julia> module polytope
           struct Polytope end
       end
Main.polytope
```

(but let's discuss this in person)